src/services/configuration.ts

```diff
@@ -0,0 +1,98 @@
+import Role from "./roles";
+
+const CONFIGURATION: AuthorizerConfig = {
```

Comment:
i will need to catch up with you on that one please when you have time but I suspect we might need some guidance from architects here.

Reply:
I've already had some guidance from @SanjeetBJSS here. We explored several options:

- Secrets Manager: this is not a secret. Also, access is restricted to very few people with access to AWS and needs to be granted explicitly for each secret - this means there is no clear definition of role behaviour accessible to the entire CVS organization, which would make debugging a lot slower. It can also take up to 100 milliseconds to load this, which adds up given that the authorizer is called by the whole of CVS.
- Local file: Lambda file-system handling is tenuous at best. Paths change on the container that Lambda runs on, e.g. `LAMBDA_TASK_ROOT` as well as an overwritten project root directory needs to be taken into account. In general, it is not good practice to access local files from within the Lambda, so we decided against it after trying it.
- Configuration in code: clearly visible and accessible to everyone, loads extremely fast, simple to understand, doesn't need to be parsed.

We went for configuration in code. But please let me know if there is a good reason to change this